orchestra/frontend/src/state/timer.ts

```diff
@@ -0,0 +1,53 @@
+import axios from 'axios'
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+
+import { AppThunk } from './store'
+
+interface Timer {
+  id: number,
+  start_time: string | null,
+  stop_time: string | null,
+  description: string | null,
+  worker: number,
+  assignment: number | null,
+  time_worked: string
+}
+
+interface TimerState {
+  status: Timer | null
+}
+
+const initialState: TimerState = {
+  status: null
+}
+
+const timer = createSlice({
```

Comment:
re: above---as an example, our timer slice doesn't live in /slices/, but also doesn't have selectors so who knows :)

Reply:
I think the timer slice should live in `/slices/`, just that we didn't get going to update it.